templates/blog/blog.html

```diff
@@ -27,8 +27,21 @@
                 {% trans time=post.publish_on|date(_("N j, Y, g:i a")) %} posted on {{ time }}{% endtrans %}
             </span>
             {% if post.is_editable_by(request.user) %}
-                <span> [<a href="{{ url('admin:judge_blogpost_change', post.id) }}">{{ _('Edit') }}</a>]</span>
+                <span> [<a href="{{ url('admin:judge_blogpost_change', post.id) }}">{{ _('Edit') }}</a>]</span>            
+            {% else %}
+                {% for author in post.authors.all() %}
```

Comment:
chỗ này check hơi nhiều for, có thể làm đơn giản hơn thế này: kiểm tra xem `post.organizations.all()` giao với `request.profile.organizations.all()` là được. 

Thêm 1 điều kiện là `request.profile` có phải là một admin trong các organization đó không. Các bước này có thể làm ở backend cho đơn giản. Kiểu như tạo một biến `context["show_org_edit"]` ở backend.

Reply:
thực ra làm trên frontend cũng ok. Với mỗi org mà mình là admin hoặc thành viên + tác giả thì hiển thị [Edit in OrgName]